code/variables.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, useJSON):  # useJSON=False): #self):
             self.MAXIMUM_DURATION = 24   # max number of frames the MCC can last for
             self.DIRS = {'mainDirStr': "../firstattempt",
                          'TRMMdirName': "../datadir/TRMM",
-                         'CEoriDirName': "../datadir/MERG"}
+                         'CEoriDirName': "../datadir/MERG",
+                         'MERGBinaryDirName': "../datadir/MERGBinary"}
```

Comment:
You shouldn't add another data source here for this. For this use case, the 'CEoriDirName': is "../datadir/MERGBinary"